conf/modules.config

```
@@ -880,20 +879,36 @@ process{
     }
 
     //CONTROLFREEC
-    withName: 'MPILEUP_NORMAL' {
-        ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}.normal" :  "${meta.id}_${intervals.simpleName}.normal" }
+    withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_MPILEUP_NORMAL:SAMTOOLS_MPILEUP' {
+        ext.prefix = { meta.num_intervals <= 1 ? "${meta.id}.normal" : "${meta.id}_${intervals.simpleName}.normal" }
+    }
+
+    withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_MPILEUP_TUMOR:SAMTOOLS_MPILEUP' {
+        ext.prefix = { meta.num_intervals <= 1 ? "${meta.id}.tumor" : "${meta.id}_${intervals.simpleName}.tumor" }
+    }
+
+    withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_MPILEUP_NORMAL:CAT_MPILEUP' {
+        ext.prefix = { "${meta.id}.normal.mpileup.gz" }
+    }
+
+    withName: 'NFCORE_SAREK:SAREK:PAIR_VARIANT_CALLING:RUN_MPILEUP_TUMOR:CAT_MPILEUP' {
+        ext.prefix = { "${meta.id}.tumor.mpileup.gz" }
     }
 
-    withName: 'MPILEUP_TUMOR' {
-        ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}.tumor" :  "${meta.id}_${intervals.simpleName}.tumor" }
+    withName: 'NFCORE_SAREK:SAREK:TUMOR_ONLY_VARIANT_CALLING:RUN_MPILEUP:CAT_MPILEUP' {
+        ext.prefix = { "${meta.id}.tumor.mpileup.gz" }
     }
 
-    withName: 'CAT_MPILEUP_NORMAL' {
-        ext.prefix = { "${meta.id}.normal.mpileup" }
+    withName: 'NFCORE_SAREK:SAREK:GERMLINE_VARIANT_CALLING:RUN_MPILEUP:SAMTOOLS_MPILEUP' {
+        ext.prefix = { meta.num_intervals <= 1 ? "${meta.id}.tumor" : "${meta.id}_${intervals.simpleName}.tumor" }
```

```suggestion
        ext.prefix = { meta.num_intervals <= 1 ? "${meta.id}.normal" : "${meta.id}_${intervals.simpleName}.normal" }
```

since this is part of the germline subworkflow